rmf_building_map_tools/pit_crew/pit_crew.py

```diff
@@ -44,6 +45,7 @@
 import io
 import os
 import re
+import subprocess
```

Comment:
Nit: can you bump this line up to near the top, since the import lines are sorted from long to short